zingolib/src/lightclient/save.rs

```diff
@@ -79,6 +85,7 @@ impl LightClient {
             .map_err(String::from)
     }
 
+    #[cfg(not(any(target_os = "ios", target_os = "android")))]
     pub(super) fn write_file_if_not_exists(dir: &Path, name: &str, bytes: &[u8]) -> io::Result<()> {
```

Comment:
this function is called in `set_sapling_params` which also calls `get_zcash_params_path` which includes code for mobile archs. this is function is therefore broken for ios/android. could you investigate?

Reply:
feature-gated that block